templates/packer-ansible.dockerfile

```
@@ -0,0 +1,16 @@
+FROM ubuntu:18.04
+
+ENV PATH=/root/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common && \
+    apt-add-repository --yes --update ppa:ansible/ansible && \
+    apt-get update && \
+    apt-get install -y ansible unzip wget python3-distutils python3-apt
```

```suggestion
    apt-get install -y ansible unzip wget python3-distutils python3-apt && \
    rm -rf /var/lib/apt/lists/*
```

adds a little bit of clean up.